docs/Project.toml

```diff
@@ -45,7 +46,8 @@ Enzyme = "0.12, 0.13"
 Flux = "0.14"
 ForwardDiff = "0.10"
 IterTools = "1"
-Lux = "1"
+MLUtils = "0.4"
+Lux = "0.5.7, 1"
```

Comment:
why pull back Lux?

Reply:
Hmm I think it showed up in a rebase conflict. Doesn't need to be held back